### PR TITLE
Updated Section article and references to Settings tree

### PR DESCRIPTION
**Pull request by eshanrnh**
## Description

While updating the Settings section in the Section article, I realized this article could use some updates since some/a lot of the below info is outdated: 

- Rewrote the content states info as it is no longer valid in v14.
- Updated descriptions of some sections to add more info.
- Removed Forms as it's not a default section in the Backoffice when CMS is installed. Instead, I added an Add-ons section.
- Help section is no longer a thing so rewrote the section to add relevant info.
- Updated references to the "Settings tree"

I will change the PR state once PR #6235 is merged since it uses the same image from the mentioned PR.

## Type of suggestion

* [ ] Typo/grammar fix
* [ ] Updated outdated content
* [ ] New content
* [x] Updates related to a new version
* [ ] Other

## Deadline (if relevant)

_When should the content be published?_

### CODE REVIEW

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 1]
```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
...
+The Help section in Umbraco provides documentation and resources to assist in understanding and effectively using the Umbraco CMS. It typically includes the following in the _Getting Started_ Dashboard:
 
-Next to the search tool, there is a help section. In the help section you can find Backoffice tours and links to Umbraco resources such as documentation and UmbracoTV.
+* **Documentation**: Comprehensive guides, tutorials, and references covering different aspects of Umbraco.
+* **Community Forums**: Access to forums where you can ask questions, share knowledge, and seek assistance from other Umbraco community members.
+* **Resources**: Stay updated with the latest news, access documentation, watch free video tutorials, and register for live demos.
+* **Training**: Learn how to effectively use Umbraco through structured courses, webinars, and hands-on tutorials designed to enhance your proficiency with the CMS.
 
-There is also a small 'user section' with shortcuts to edit the currently logged in user, and view their most recent activities.
+Overall, the Help section serves as a valuable resource hub in navigating and leveraging the capabilities of the Umbraco CMS effectively.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:96]:
```suggestion
The Help section serves as a valuable resource hub in navigating and leveraging the capabilities of the Umbraco CMS effectively.
```

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 2]
```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
...
+## Translation
 
-You can install Umbraco Forms directly from the Backoffice by clicking the install button. Once installed, this section is where you create and manage your forms.
+The Translation section is where you create and manage Dictionary Items. By managing these dictionary items, you can ensure consistent and efficient content translation and maintenance across different languages.
 
-## Translation
+## Add-On Sections
 
-This is the section where you create and manage your dictionary items.
+You can enhance Umbraco's functionality with plugins and extensions tailored to specific needs, expanding capabilities beyond core features. Currently, Umbraco supports add-ons like Forms, Deploy, Workflow, Commerce, and UI Builder.
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:85]: How about a link to the Marketplace here?

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 3]
```diff
@@ -6,30 +6,32 @@ description: >-
...
 ## Content
 
-The Content section contains the content of the website. Content is displayed as nodes in the content tree. Nodes can also show content state:
+The Content section contains the content of the website. Content is displayed as nodes in the content tree.
+
+Nodes in Umbraco can display the following content states:
 
-* Grayed out nodes have not been published
-* <img src="images/has-unpublished-version.svg" alt="" data-size="line"> Nodes have unpublished versions (but are currently published)
-* <img src="images/protected.svg" alt="" data-size="line"> Nodes are protected from the public (can only be accessed by a logged in Member)
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:24]: Is it no longer possible to restrict access to certain content?

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 4]
```diff
@@ -6,30 +6,32 @@ description: >-
...
-* <img src="images/locked.svg" alt="" data-size="line"> Nodes are currently locked/non-deletable
-* <img src="images/is-container.svg" alt="" data-size="line"> Nodes are containers (a List View for example)
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:26]: This is still a state that a piece of content can be in, right?

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 5]
```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
 * Extension Insights
 * Webhooks
 
-The **Settings** section of the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
+The **Settings** section in the Umbraco backoffice has its own set of default dashboards. For more information, see the [Settings Dashboards](settings-dashboards.md) article.
 
 ## Packages
 
 In this section, you can browse and install packages into your Umbraco solution. You can also get an overview of all the installed packages as well as uninstall packages you no longer need.
```
Comment (sofietoft) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:69]: You can't install packages directly from this section, so I think we should take that out of this section!

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks good @eshanrnh !

I've added some comments - please take a look when you can.

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 4 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:26]: No, the content node doesn't have different states/symbols on it anymore.

Image for reference: 
image

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 3 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:24]: I don't see a lock option any where 🙈

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 5 above)
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:69]:
```suggestion
In this section, you can browse the different packages available for your Umbraco solution. You can also get an overview of all the packages you have installed or created.
```

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 6]
```diff
@@ -60,35 +62,38 @@ The Settings section consists of:
...
+## Translation
 
-You can install Umbraco Forms directly from the Backoffice by clicking the install button. Once installed, this section is where you create and manage your forms.
+The Translation section is where you create and manage Dictionary Items. By managing these dictionary items, you can ensure consistent and efficient content translation and maintenance across different languages.
 
-## Translation
+## Add-On Sections
 
-This is the section where you create and manage your dictionary items.
+You can enhance Umbraco's functionality with plugins and extensions tailored to specific needs, expanding capabilities beyond core features. Currently, Umbraco supports add-ons like Forms, Deploy, Workflow, Commerce, and UI Builder.
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/backoffice/sections.md:85]:
```suggestion
You can enhance Umbraco's functionality with plugins and extensions tailored to specific needs, expanding capabilities beyond core features. Currently, Umbraco supports add-ons like Forms, Deploy, Workflow, Commerce, and UI Builder. You can find the different packages at [Umbraco Marketplace](https://marketplace.umbraco.com/).
```

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 2 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md]: Added a link to the marketplace

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 5 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md]: Updated the text.

**Comment by eshanrnh** [comment]
I've incorporated your comments, @sofietoft 🙌 I accidently pushed some more updated articles to this PR rather than creating a new PR. They are very small updates. In case, if it's too much let me know and I'll create  another PR for those changes.

File: 14/umbraco-cms/fundamentals/data/relations.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ To create and use Relations in your Umbraco Website, you need to define a 'Relat
 
 ### Create a Relation Type
```
Comment (eshanrnh) [inline 14/umbraco-cms/fundamentals/data/relations.md:35]: This section needs to be updated at some point. You might see some outdated images here. Currently, the Create button is missing from the UI (open issue in CMS) and hence the section cannot be updated. I've added a task so we can track it later.

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 3 above)
Comment (sofietoft) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:24]: It's the "Pubic access" option, where you can limit access to certain members or member groups.
I'm pretty sure this is still an option! See: https://docs.umbraco.com/umbraco-cms/tutorials/members-registration-and-login#member-only-pages-restricted-access

**Review by sofietoft — CHANGES_REQUESTED** [review]
Looks good  @eshanrnh ! 💪 

I've added some comments here and there.
Let me know if something doesn't make sense!

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 8]
```diff
@@ -60,35 +66,50 @@ The Settings section consists of:
...
-Manage, create, and customize members and member groups.
+## Members
 
-## Forms
+The Members section allows to create and manage member profiles, set up member groups, and control Member's access to restricted content on the website.
 
-You can install Umbraco Forms directly from the Backoffice by clicking the install button. Once installed, this section is where you create and manage your forms.
+For more information, see the [Members](../../fundamentals/data/members.md) article.
 
 ## Translation
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:91]: yup, that's right.

File: 14/umbraco-cms/fundamentals/backoffice/sections.md (hunk 3 above)
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:24]: Got it..

File: 14/umbraco-cms/fundamentals/backoffice/sections.md [hunk 9]
```diff
@@ -60,35 +66,50 @@ The Settings section consists of:
...
 
-This is the section where you create and manage your dictionary items.
+The Translation section is where you create and manage Dictionary Items. By managing these dictionary items, you can ensure consistent and efficient content translation and maintenance across different languages.
+
+For more information, see the [Dictionary Items](../../fundamentals/data/dictionary-items.md) article.
+
+## Add-On Sections
```
[the remark this answers is not in the document]
Comment (eshanrnh) [reply 14/umbraco-cms/fundamentals/backoffice/sections.md:93]: Sure, added it..

**Comment by eshanrnh** [comment]
Thanks, @sofietoft for your review 🙌 I've incorporated your comments. Let me know what you think 💪

**Review by sofietoft — APPROVED** [review]
Thanks for considering the suggestions!

Look great 💪 
Ready to merge.

**Pull request merged**